conodictor/2.3.1/Dockerfile

```diff
@@ -0,0 +1,77 @@
+# base image
+FROM ubuntu:20.04
+
+# metadata
+LABEL base_image="ubuntu:20.04"
+LABEL version="2"
```

Comment:
Version is 1 here as it is first version of this dockerfile.

Version is a way to track changes on a specific dockerfile (upstream is the same but we patch build rule for example)